Bottom: 956379d662e5140bff631becfdb3af8e5cceb981
Top:    bf45407945a0290ab87ec31f1cb51083c07120ee
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 12:42:05 +0000

Add missing return type to cvc_list_dump (TODO: Roll into source commit)


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 72fca37..364d472 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -511,6 +511,7 @@ next_cvc_from_other_poly (CVCList *start)
   return l;
 }
 
+static void
 cvc_list_dump (CVCList *list)
 {
   VNODE *node;
